src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -682,6 +895,43 @@ namespace Microsoft.FSharp.Collections
         /// <returns>The final state value.</returns>
         ///
         /// <exception cref="T:System.ArgumentNullException">Thrown when the either of the input sequences is null.</exception>
+        ///
+        /// <remarks>
+        /// TODO: Shouldn't this have a remark about consuming the whole sequence(s) just like foldBack?
```

Comment:
Put this here to not forget to ask @dsyme about it.

Reply:
Yes it should, thanks

Reply:
Done.